### ISSUE TITLE
Extension autoopens when my debugger starts even on non ionic projects

### ISSUE THREAD

**Issue by devo-devo** [ionic-team/vscode-extension#160]
Is there anyway to disable this setting?

**Comment by dtarnawsky** [comment]
Hey @devo-devo , do you have an example project that this is opening for?
(I can fix this quickly if I have a sample to work with)

**Comment by devo-devo** [comment]
thanks for the quick reply @dtarnawsky!

Here's an example repo: https://github.com/devo-devo/django-example

I think all that needs to be done is set up a python environment and a pip install. Then try running the debugger and if you have the ionic extension installed the sidebar will open...

```console
# Linux
sudo apt-get install python3-venv    # If needed
python3 -m venv .venv
source .venv/bin/activate

# macOS
python3 -m venv .venv
source .venv/bin/activate

# Windows
py -3 -m venv .venv
.venv\scripts\activate
```
Then:
```
python -m pip install --upgrade pip
python -m pip install django
```



https://code.visualstudio.com/docs/python/tutorial-django

**Comment by dtarnawsky** [comment]
Thanks @devo-devo , I was able to reproduce and fix the issue. This should be available in the upcoming release (hopefully tomorrow).

**Comment by dtarnawsky** [comment]
Hey @devo-devo , this issue has been resolved in v1.80. Thanks for the help with the reproduction.
